Gemfile.lock

```diff
@@ -234,7 +234,7 @@ GEM
       activemodel (~> 6.0)
       activesupport (~> 6.0)
       railties (~> 6.0)
-    metasploit-credential (5.0.7)
+    metasploit-credential (5.0.8)
```

Comment:
Adds support for OpenSSL3 https://github.com/rapid7/metasploit-credential/pull/163